### PR TITLE
Change README's link from github.com/bbatsov to github.com/rubocop-hq

### PR DESCRIPTION
**Pull request by onk**
tl:dr

As a similar topic, Is it possible to transfer the followings to rubocop-hq?

* AppVeyor
* Code Climate
* Inch CI
* Gitter

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [ ] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [ ] Run `rake default` or `rake parallel`. It executes all tests and RuboCop for itself, and generates the documentation.

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by bbatsov** [comment]
Thanks!

> As a similar topic, Is it possible to transfer the followings to rubocop-hq?

I think I already did this for AppVeyor, have to check the others. I haven't used Inch CI and Gitter in ages, so I can just remove them as well. Not sure if the other contributors find them useful.
